.github/workflows/ci.yml

```diff
@@ -1034,7 +1030,7 @@ jobs:
                     # Check if this repository has a mapping against arbitrary CMS majors.
                     # This is represented by a major version mapping of "*" to either an empty array, or an array with the current branch in it.
                     SUPPORTED_BRANCHES=$(jq -r --arg repo ${GITHUB_REPOSITORY}  '.[] | map(select(.github == $repo))[].majorVersionMapping | select(has("*"))[]' __modules.json)
-                    if [[ $SUPPORTED_BRANCHES == "[]" || $(echo '["5"]' | jq -r --arg branch ${BRANCH_MAJOR} 'index($branch)') != "null" ]]; then
+                    if [[ $SUPPORTED_BRANCHES == "[]" || $(echo '["6"]' | jq -r --arg branch ${BRANCH_MAJOR} 'index($branch)') != "null" ]]; then
```

Comment:
I'm honestly not 100% sure what this was doing... but `5` does seem to represent "the highest stable major release line" here. If I'm right about that, then this should indeed be updated.